wolox-books/src/app/store/books.reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { Action } from '@ngrx/store'
+import { Book } from '../models/lists-books.model'
+import * as ListBooksCarActions from './books.actions'
+
+const initialState: Books = { Books: []}
+
```

Comment:
initial State shouldn't be of type books you should create an interface that declares the type of the initialState